wavefront_python_sdk/common/connection_handler.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import, division, print_function
+
+from wavefront_python_sdk.common.utils import AtomicCounter
```

Comment:
It's generally considered a good practice to use imported classes from their modules' namespaces rather than importing them into your global namespace, to avoid name clashes.